### DISCUSSION TITLE
GitHub sorting within search not working anymore

### DISCUSSION THREAD

**Discussion by Hakky54** [community/community#20099] [Other Feature Feedback, Questions, & Ideas]
I initially mentioned this issue here https://github.community/t/github-ordering-within-search-doesnt-work/259081 but opened a discussion here after the suggestion of the guy who replied to my issue.

I noticed that sorting after searching for something on GitHub does not work anymore. It was working somewhere mid June. For example if i search for my own username and select the code tab i will get the best match as shown here https://github.com/search?q=Hakky54&type=code after selecting recently indexed from the drop-down box for sorting nothing happens anymore for mobile. Also other options doesn't work anymore such as language etc. Any idea what could be the cause of it?

**Comment by arkivanov** [comment]
Same issue for me, but only on mobile web version. Sorting works fine on the desktop version.

**Reply by masterscup** [reply]
Same issue for me. Also sorting seems not working as it used to on the desktop version, seems latest commits or revisions no longer will be returned.

**Comment by ohmybahgosh** [comment]
Both the latest chrome stable & dev android versions will not update after selecting the dropdown sort option unless I switch to "Desktop Mode"

**Reply by rodber** [reply]
same issue

**Comment by jfullstackdev** [comment]
there is a new search from GitHub 

https://cs.github.com/about

although this one is still receiving so many bug reports, but you can try them too

**Reply by rodber** [reply]
I'm already in the preview, it looks neat! Too bad for me that filtering is only for language. I'm interested in more filtering options.

**Reply by Hakky54** [reply]
This is a nice addition, thank you for sharing it. However it would be nice if the bug would be fixed on mobile. The new search is still limited in features

**Reply by jfullstackdev** [reply]
yup, as always we know that they constantly update, it's worth the wait

**Comment by Hakky54** [comment]
It seems like this sorting is working again on mobile devices

**Reply by arkivanov** [reply]
Yes, good news!

**Reply by UmerYousuf** [reply]
What do you mean is it working on mobile? How?

**Comment by Hakky54** [comment]
It seems like the issue is back. With the new search being introduced the sorting on recent is missing when I search on something and select the code section

**Reply by masterscup** [reply]
trying new search and found recent indexed missing. any ideas how to sort code search results using new search? Also it seems that you can only review 5 pages of search results. anyone having the same problems?

**Comment by UmerYousuf** [comment]
Is there any way to increase pagination results and enable sorting?

**Comment by Hakky54** [comment]
Any idea whether sorting `most recent` or `recently indexed` will come back when searching through the code? In the new search it is still not there and the old search is not available anymore... I was using that option daily, so I am still hoping it will come back soon in the new search solution. Hopefully the developers of the search api and UI will notice this discussion here.

**Reply by uBadRequest** [reply]
I was just looking for a way to go back :(

**Comment by sudoshindo** [comment]
Bring the code result sorting back.

STOP TAKING AWAY GOOD FEATURES.
